reference/rpc/PANIC_EXIT.md

```
@@ -0,0 +1,50 @@
+# `panic_exit` – Emergency flat position
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)